plugins/other/pacman

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# DESCRIPTION
+#
+# This plugin monitors the number of pending updates in pacman.
+#
+# WARNINGS
+#
+# This plugin will issue a warning state whenever any package can be updated.
+#
+# ENVIRONMENT
+#
+# env.WARNING The number of updates at which a warning will be issued. If not
```

Comment:
Yes, I share your style preference.

There is no explicit style definition for munin - I just ran a quick "survey" over the plugins of this contrib repository (looking for the most common documentation style):
```
$ grep -Frh "env." plugins/ | sed -n '/^.*env\.[a-z]/p' | wc -l
1734
$ grep -Frh "env." plugins/ | sed -n '/^.*env\.[A-Z]/p' | wc -l
107
```
Thus, you may feel free to stick to the ALL_CAPS style.